MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -355,6 +375,50 @@ protected virtual void FetchDataFromXmlNode(XmlReader reader, MetadataResult<T>
                         break;
                     }
 
+                case "watched":
+                    {
+                        var val = reader.ReadElementContentAsBoolean();
+
+                        if (userData != null)
+                        {
+                            userData.Played = val;
+                        }
+
+                        break;
+                    }
+
+                case "playcount":
+                    {
+                        var val = reader.ReadElementContentAsString();
+                        if (!string.IsNullOrWhiteSpace(val) && userData != null)
+                        {
+                            if (int.TryParse(val.Split(' ')[0], NumberStyles.Integer, UsCulture, out var count))
```

Comment:
Do some playcounts have spaces in the value?

Reply:
I hope they don't. Seems like I have copied from runtime which can contain spaces.